src/index.html

```diff
@@ -1,13 +1,50 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0,
+minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="product" data-qa="card">
+      <img src="/images/imac.jpeg"
+        alt="photo-screen"
+        class="product__img"
+      >
+      <div class="product__text">
+        <h3 class="product__title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h3>
+        <div class="product__code">
+          Код товара: 195434
+        </div>
+        <div class="product__rating">
+          <div class="product__stars stars--4">
```

Comment:
for now `4` don't mean anything. this stars should work the same like in `stars`  task. If I change 4 to 3 - there should be only 3 active stars

Reply:
use nth-child(-n+`digit`) like you did it before

Reply:
sorry, I  didn't understand this right at the first time. I hope this is going to work in a better way after changes I made.  But if my code still doesn't work in the way it is supposed to do, I'll be glad to make changes again.